Automattic/wp-calypso
# Apply new design for the core profiler Jetpack connect page

Related to #92103 

This PR applies the new design for the "One last step!" page for both singed in & out.

This PR is going to serve as a base PR for the upcoming PRs for https://github.com/Automattic/wp-calypso/issues/92105

## Proposed Changes

Signed-in:

![Screenshot 2024-07-31 at 2 02 12 PM](https://github.com/user-attachments/assets/2e66058c-2f40-4509-9ce7-7de0183e6e49)

Signed-out:
![Screenshot 2024-07-31 at 2 02 03 PM](https://github.com/user-attachments/assets/43f4ef58-7477-436f-a027-6bccc60a3cb3)



## Why are these changes being made?


* Apply new design

## Testing Instructions

* Create a new JN site with the latest trunk
* Start the core profiler and choose 
* Make sure to check Jetpack on the extensions page and click on `Continue`. You'll be redirected to the Jetpack connect page.
* Copy the URL on the Jetpack connect page.
* Replace https://wordpress.com with your local Calypso url (http://calypso.localhost:3000/)
* Paste the URL.
* You should see the updated design.
* Open an incognito window and past the URL.
* You should see the updated design for signed-out users.

## Pre-merge Checklist
